app/models/sample.rb

```diff
@@ -130,6 +130,10 @@ def related_organism_ids
     organism_ids | ncbi_linked_organisms.map(&:id)
   end
 
+	def get_value(attribute)
```

Comment:
this isn't needed, should use `get_attribute_value(attr)`

Reply:
It's been fixed.